pkg/cmd/roachtest/tests/tpcc.go

```diff
@@ -1346,9 +1350,11 @@ func registerTPCC(r registry.Registry) {
 		LoadWarehousesGCE:   1000,
 		LoadWarehousesAWS:   1000,
 		LoadWarehousesAzure: 1000,
+		LoadWarehousesIBM:   1000,
 		EstimatedMaxGCE:     750,
 		EstimatedMaxAWS:     900,
 		EstimatedMaxAzure:   900,
+		EstimatedMaxIBM:     900,
```

Comment:
I'd imagine these numbers to be lower based on my preliminary benchmarking [1]. We can tweak them later; the only down-side is the search will take more iterations.

[1] https://gist.github.com/srosenberg/4f5bda2fda51978923981a7f55df4933

Reply:
I did copy/paste the AWS and Azure values. Agreed we'll have to tweak later.